manifests/v/Vup/Vup/0.13.3/Vup.Vup.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Vup.Vup
+PackageVersion: 0.13.3
+Installers:
+- InstallerUrl: https://github.com/vup-app/vup/releases/download/v0.13.3/VupSetup-0.13.3-BETA-Windows.exe
+  Architecture: x64
+  InstallerType: inno
+  InstallerSha256: 32D3A6D1AF03B67406E0CB4A91962479ADADD9F6A91B42A6318562A49A367B4A

```

Comment:
```suggestion
  InstallerSha256: 32D3A6D1AF03B67406E0CB4A91962479ADADD9F6A91B42A6318562A49A367B4A
  InstallerSwitches:
    Custom: /AllUsers
```

Reply:
Will this fix the installer failing without user input? The only two things that stop it on my local machine are windows defender and choosing a user. I presume the /AllUsers fixes the second one.